docs/SUMMARY.md

```diff
@@ -26,6 +26,7 @@
   * [Config Maps](user-guide/creating-application/config-maps.md)
   * [Secrets](user-guide/creating-application/secrets.md)
   * [Environment Overrides](user-guide/creating-application/environment-overrides.md)
+  * [Job And CronJob](user-guide/creating-application/job-and-cronjob.md)
```

Comment:
Remove this, this will be included as a type of chart separately.

Reply:
the requested changes has been made.